examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,72 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+minter.setDebugLabel('Minter');
+receiver.setDebugLabel('Receiver');
+
+const gasLimit = 5000000;
+if (stdlib.connector != 'ALGO') {
+    minter.setGasLimit(gasLimit) && receiver.setGasLimit(gasLimit)
+};
+
+const mintAddr = minter.getAddress();
+console.log(`Minter's address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`Receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who, tok) => tok ? (await stdlib.balanceOf(who, tok)) : fmt(await stdlib.balanceOf(who));
+
+const logBalance = async (acc, tok) => {
+    const bal = await getBal(acc, tok);
+    const unit = tok ? 'of the NFT' : stdlib.standardUnit;
+    console.log(`${acc.getDebugLabel()} has ${bal} ${unit}.`);
+    return bal;
+}
+
+await logBalance(minter);
+
+const name = (stdlib.connector == 'ALGO') ? "JPAlgos" : "JPals";
+const symbol = (stdlib.connector == 'ALGO') ? "JPA" : "JPAL";
+
+const opts = { 
+    supply: 1, 
+    url: "ipfs://bafybeigdyrzt5...", //asset url
```

Comment:
I think it is more compelling if you demonstrate actually creating it too. The same information that you're going to put in the Note field, you want to upload to IPFS.

This page has a little tutorial:

https://js.ipfs.tech/

under "Adding data to IPFS"

If you are nervous about including it in the code always or having our CI do it, etc, then at least make a little program on the side (or a way to run this program) and comment on it in your guide